flytekit/extras/tasks/shell.py

```diff
@@ -184,6 +184,20 @@ def script(self) -> typing.Optional[str]:
     def script_file(self) -> typing.Optional[os.PathLike]:
         return self._script_file
 
+    def make_export_string_from_env_dict(self, d) -> str:
```

Comment:
can we have type for d
```suggestion
    def make_export_string_from_env_dict(self, d: str) -> str:
```

Reply:
Done, `d` is `typing.Dict`, can enforce the keys and values to be `str, str`, but technically, as long as the key-value can be cast to str, it should be ok.